### PR TITLE
Add cloud provider outage notifications

### PR DESCRIPTION
**Pull request by dustman9000**
Add messaging around cloud provider outages.

### CODE REVIEW

**Comment by iamkirkbater** [comment]
This looks fine to me, but I'll let others comment as well.

File: osd/gcp/GCP_outage.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Info",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "GCP Outage Affecting Connectivity",
+    "description": "Your cluster is currently being affected by a GCP outage, which includes control plane and workload connectivity. To stay up to date with the current AWS status update, please follow https://status.cloud.google.com/",
```
Comment (jaybeeunix) [inline osd/gcp/GCP_outage.json:6]: you gotta' lil' aws in there

**Comment by tnierman** [comment]
Seems like the messages could be combined & a variable used for specifying the cloud-provider. Is the goal to just limit the number of `-p` flags you have to use?

**Comment by clcollins** [comment]
Looks good to me too

**Pull request merged**